src/site/components/up_to_top_button.html

```diff
@@ -12,16 +12,13 @@
 
 {% endcomment %}
 
-<div data-template="components/up_to_top_button" id="top-button-container" class="vsa-top-button-container">
+<div class="vsa-top-button-container" data-template="components/up_to_top_button" id="top-button-container">
     <div class="usa-grid usa-grid-full">
-        <div id="usa-width-container" class="usa-width-three-fourths">
```

Comment:
Do you know why this was an `id` to begin with? CC @zacharymorel Just want to make sure it's not being targeted by JS

Reply:
Ah, nvm I see now